packages/ember-htmlbars/tests/integration/attrs_lookup_test.js

```diff
@@ -1,154 +0,0 @@
-
-var owner, view;
-
-import isEnabled from 'ember-metal/features';
-if (!isEnabled('ember-glimmer')) {
-  // jscs:disable
-
-QUnit.module('component - attrs lookup', {
-  setup() {
-    owner = buildOwner();
-    owner.registerOptionsForType('component', { singleton: false });
-    owner.registerOptionsForType('view', { singleton: false });
-    owner.registerOptionsForType('template', { instantiate: false });
-    owner.register('component-lookup:main', ComponentLookup);
-  },
-
-  teardown() {
-    runDestroy(owner);
-    runDestroy(view);
-    owner = view = null;
-  }
-});
-
-QUnit.test('should be able to lookup attrs without `attrs.` - template access', function() {
-  owner.register('template:components/foo-bar', compile('{{first}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{foo-bar first="first attr"}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(view.$().text(), 'first attr');
-});
-
-QUnit.test('should be able to lookup attrs without `attrs.` - component access', function() {
-  var component;
-
-  owner.register('component:foo-bar', Component.extend({
-    init() {
-      this._super(...arguments);
-      component = this;
-    }
-  }));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{foo-bar first="first attr"}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(component.get('first'), 'first attr');
-});
-
-QUnit.test('should be able to modify a provided attr into local state #11571 / #11559', function() {
-  var component;
-
-  owner.register('component:foo-bar', Component.extend({
-    init() {
-      this._super(...arguments);
-      component = this;
-    },
-
-    didReceiveAttrs() {
-      this.set('first', this.getAttr('first').toUpperCase());
-    }
-  }));
-  owner.register('template:components/foo-bar', compile('{{first}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{foo-bar first="first attr"}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(view.$().text(), 'FIRST ATTR', 'template lookup uses local state');
-  equal(component.get('first'), 'FIRST ATTR', 'component lookup uses local state');
-});
-
-QUnit.test('should be able to access unspecified attr #12035', function() {
-  var component;
-
-  owner.register('component:foo-bar', Component.extend({
-    init() {
-      this._super(...arguments);
-      component = this;
-    },
-
-    didReceiveAttrs() {
-      equal(this.get('woot'), 'yes', 'found attr in didReceiveAttrs');
-    }
-  }));
-  // owner.register('template:components/foo-bar', compile('{{first}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{foo-bar woot="yes"}}')
-  }).create();
-
-  runAppend(view);
-
-  // equal(view.$().text(), 'FIRST ATTR', 'template lookup uses local state');
-  equal(component.get('woot'), 'yes', 'component found attr');
-});
-
-QUnit.test('should not need to call _super in `didReceiveAttrs` (GH #11992)', function() {
```

Comment:
Ahh ok, I think I must have spaced out over the `get` vs `getAttr` comparison. Will port.